wikidata_game/api.php

```diff
@@ -0,0 +1,184 @@
+<?PHP
+// This is a POC, don't judge
+function getDb() {
+    $dbmycnf = parse_ini_file("../replica.my.cnf");
+    $dbuser = $dbmycnf['user'];
+    $dbpass = $dbmycnf['password'];
+    $dbhost = "tools.db.svc.eqiad.wmflabs";
+    $dbname = "s54377__wd_ref_island_p";
+    return new PDO('mysql:host=' . $dbhost . ';dbname=' . $dbname . ';charset=utf8', $dbuser, $dbpass);
+}
+
+function getDesc() {
+    $title = "Reference hunt!";
+    return [
+        "label" => ["en" => $title],
+        "description" => ["en" => "[https://www.wikidata.org/wiki/Wikidata:Automated_finding_references_input Reference hunt] suggestions to add references in items based on structued data in the web. Source code can be found in [https://github.com/wmde/reference-island here]"],
```

Comment:
Should the links be in square brackets?

Reply:
It works like this too, look at Kian (https://tools.wmflabs.org/wikidata-game/distributed/#game=15) and its API response (https://kian.toolforge.org/api.php?action=desc&callback=bar)

Reply:
Great! It looked off in the "try your api mode" but I guess that's just a rough edge in the magnus code